app/js/overview.js

```diff
@@ -27,7 +27,7 @@ document.querySelector(".overview").addEventListener("click", function() {
 })
 
 document.querySelector('table').addEventListener('click', function(e) {
-    current = e.target.parentNode.dataset.id
+    current = e.target.parentNode.dataset.questionOrderId
```

Comment:
This breaks the overview page in that, you can no longer click the question and be taken to that question back in the question page